File: src/main/java/com/sfh/pokeRogueBot/model/modifier/impl/PokemonPpRestoreModifierItem.java
package com.sfh.pokeRogueBot.model.modifier.impl;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class PokemonPpRestoreModifierItem extends PokemonMoveModifierItem {

    public static final String TARGET = "PokemonPpRestoreModifierType";

    private int restorePoints;

}

File: src/main/kotlin/com/sfh/pokeRogueBot/model/modifier/impl/PokemonPpRestoreModifierItem.kt
package com.sfh.pokeRogueBot.model.modifier.impl

class PokemonPpRestoreModifierItem : PokemonMoveModifierItem() {

    companion object {
        const val TARGET = "PokemonPpRestoreModifierType"
    }

    var restorePoints: Int = 0
}